WBooks/Screens/Library/CustomBookCell/CustomBookCell.swift

```diff
@@ -30,7 +30,11 @@ class CustomBookCell: UITableViewCell, NibLoadable {
     }
     
     func setBookAttributes(cellVM: CustomBookCellViewModel) {
-        imageSource.image = cellVM.image
+        if cellVM.image != nil {
+            imageSource.loadUrl(from: cellVM.image ?? "")
+        } else {
+            imageSource.image = UIImage.defaultBook
+        }
```

Comment:
you could validate `cellVM.image` too like `if let image = cellVM.image, image.isNotEmpty { ... } `

Reply:
sorry, you are validating `.isNotEmpty` in viewModel, so you could do just `if let image = cellVM.image { ..`

Reply:
Also, if you get an error downloading the image it is going to be empty, so you could assign `UIImage.defaultBook` always as default, and then ask if it has a image to download `if let image = cellVM.image {..`

Reply:
nice